CONTRIBUTING.md

```diff
@@ -44,6 +44,25 @@ Build logs are written to stdout and artifacts written to the current directory.
 When new versions of the packages are released, they should be included in [CHaP](https://github.com/input-output-hk/cardano-haskell-packages).
 See the CHaP README for [instructions](https://github.com/input-output-hk/cardano-haskell-packages#-from-github).
 
+## Release Branches
+
+When needed we use release branches: `release/cardano-node-*`, e.g.
+`release/cardano-node-1.35.x` (which ought to be simply named as
+`release/cardano-node-1.35`).
+
+Note that `CHaP` allows us to make releases of packages independently of each
+other (especailly non breaking changes), so there might be other release
+branches, e.g.  `release/network-mux-*`.  They MUST follow the following
+nameing convention: `release/${package-name}-${version}`.
```

Comment:
```suggestion
naming convention: `release/${package-name}-${version}`.
```

Reply:
Should there be a relationship between the node release and the package name? I guess not, but maybe be better to clarify.

Reply:
Do you mean node release and branch name?

I wanted to leave us the room to use `release/cardano-node-{node-version}` branches and `release/{package}-{version}`.  Both might be handy at times.

Reply:
> Do you mean node release and branch name?

Hehe, no, I meant the name of the branches in `ouroboros-network`, and name of the branch in `cardano-node`. For instance, for release `1.36` of Cardano, how should we name our branches? Something like: 

- `cardano-node@release/cardano-node-1.36`
- `ouroboros-network@release/ouroboros-network-1.36`

?

Reply:
As we do now: `release/cardano-node-x` so the `release` branch rules apply.